code/modules/mob/living/carbon/human/human.dm

```diff
@@ -184,6 +184,10 @@
 		dat += "&nbsp;<A href='?src=[REF(src)];pockets=right'>[(r_store && !(r_store.item_flags & ABSTRACT)) ? "Right (Full)" : "<font color=grey>Right (Empty)</font>"]</A></td></tr>"
 		dat += "<tr><td>&nbsp;&#8627;<B>ID:</B></td><td><A href='?src=[REF(src)];item=[SLOT_WEAR_ID]'>[(wear_id && !(wear_id.item_flags & ABSTRACT)) ? wear_id : "<font color=grey>Empty</font>"]</A></td></tr>"
 
+	for (var/obj/item/bodypart/org in bodyparts)
+		if (org.bandaged)
+			dat += "<tr><td><i>[org.name]</i> wrapped with:</td><td><a href='byond://?src=\ref[src];unwrap=\ref[org.bandaged]'>[org.bandaged]</a></td></tr>" // yogs - show bandaged parts, side note: what the fuck is this proc?
+		

```

Comment:
muh modular

Reply:
How would I make this modular?  It has to be in this spot or else it does other shit.  Honestly don't know.